gf-webapp-admin/src/component/rooms/Rooms.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect, useState} from "react";
+import {IRoom} from "../../model/room";
+import {RoomsApi} from "../../../autogenerated/gf-server-api/apis/RoomsApi";
+import styles from "./rooms.module.scss";
+import {Room} from "./room/Room";
+
+export const Rooms = () => {
+  const roomsApi = new RoomsApi;
+  const [rooms, setRooms] = useState<IRoom[]>([]);
+
+  const getRooms = async () => {
+    const res = await roomsApi.apiRoomsGet();
+    setRooms(res);
+  };
+
+  useEffect(() => {
+    getRooms();
+  }, []);
+
+  return (
+    <div className={styles.rooms}>
+      {rooms.map(room => (
```

Comment:
too nested logic. Move the nested arrow function to another place

Reply:
Done